src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -17,99 +18,102 @@
         try (Connection connection = ConnectionUtil.getConnection();
-                PreparedStatement preparedStatement
-                        = connection.prepareStatement(query)) {
-            preparedStatement.setString(1, manufacturer.getName());
-            preparedStatement.setString(2, manufacturer.getCountry());
-            preparedStatement.setLong(3, manufacturer.getId());
-            preparedStatement.executeUpdate();
-            return manufacturer;
-        } catch (SQLException throwable) {
-            throw new DataProcessingException("Couldn't update a manufacturer "
-                    + manufacturer + " ", throwable);
+                PreparedStatement createStatement =
+                        connection.prepareStatement(updateRequest)) {
+            createStatement.setString(1, manufacturer.getName());
+            createStatement.setString(2, manufacturer.getCountry());
+            createStatement.setLong(3, manufacturer.getId());
+            if (createStatement.executeUpdate() > 0) {
+                return manufacturer;
+            }
+            throw new RuntimeException("No such manufacturer " + manufacturer.getId());
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t update manufacturer for name: "
+                    + manufacturer.getName() + " from DB", e);
         }
     }
 
     @Override
     public boolean delete(Long id) {
-        String query = "UPDATE manufacturers SET deleted = TRUE WHERE manufacturer_id = ?";
+        String deleteRequest = "UPDATE manufacturers SET is_deleted = true where id = ?";
         try (Connection connection = ConnectionUtil.getConnection();
-                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
-            preparedStatement.setLong(1, id);
-            return preparedStatement.executeUpdate() > 0;
-        } catch (SQLException throwable) {
-            throw new DataProcessingException("Couldn't delete a manufacturer by id " + id + " ",
-                    throwable);
+                PreparedStatement createStatement =
+                        connection.prepareStatement(deleteRequest)) {
+            createStatement.setLong(1, id);
+            return createStatement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t delete manufacturer for id: "
+                    + id + " from DB", e);
         }
     }
 
-    private Manufacturer getManufacturer(ResultSet resultSet) throws SQLException {
-        Long newId = resultSet.getObject("manufacturer_id", Long.class);
-        String name = resultSet.getString("manufacturer_name");
-        String country = resultSet.getString("manufacturer_country");
-        Manufacturer manufacturer = new Manufacturer(name, country);
-        manufacturer.setId(newId);
+    public Manufacturer setValuesFromDB(ResultSet resultSet) throws SQLException {
```

Comment:
the previous method name is more informative - let's return it back

Reply:
done